src/components/spacings/stack/stack.js

```diff
@@ -13,12 +13,29 @@ Stack.displayName = 'Stack';
 Stack.propTypes = {
   scale: PropTypes.oneOf(['xs', 's', 'm', 'l', 'xl']),
   children: PropTypes.node,
-  alignItems: PropTypes.oneOf(['stretch', 'flexStart', 'flexEnd', 'center']),
+  alignItems: PropTypes.oneOf([
+    'stretch',
+    'flex-start',
+    'flex-end',
+    'center',
+    // Deprecated
+    'flexStart',
```

Comment:
Should we `warning` when they're used?

Reply:
Hmm that might end up with a lot of noise. We can maybe do that in the next major, then in the major after that we can remove them completely.
So a two-step deprecation process.